x-pack/plugins/enterprise_search/cypress/support/e2e.ts

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Should we delete this given that it's commented out?

Reply:
I can delete the contents instead, It is expected to be there by cypress unless it is set up in a different way. And we will need it in a considerably short amount of time.

Reply:
If we need it soon, keeping it commented out is probably fine.